database/dbConnect.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+const config = require('config');
+
+const URI = `mongodb://${config.db.host}:${config.db.port}/${config.db.database}`;
+
+mongoose.set('useCreateIndex', true);
+
+module.exports = mongoose.createConnection(URI,
+    {
+        useNewUrlParser: true, useFindAndModify: false, useUnifiedTopology: true,
+    },
+    () => console.log('Connect to widgets MongoDB successful!'));
```

Comment:
what is it!?!? I said, connection to mongo DB only at api, in extreme cases, at this service you can connect radishes